src/raftstore/store/fsm/batch.rs

```diff
@@ -252,6 +252,9 @@ pub trait PollHandler<N, C> {
 
     /// This function is called at the end of every round.
     fn end(&mut self, batch: &mut [Box<N>]);
+
+    /// This function is called when batch system is going to sleep.
+    fn pause(&mut self) {}
```

Comment:
I prefer name it yield.

Reply:
Why? `PollHandler` can't do the yield job. And in my opinion, yield usually means short switch.